src/README.md

```diff
@@ -8,7 +8,7 @@ as registry.access.redhat.com/rhscl/python-{{ spec.short_ver }}-rhel7
 and the RHEL8 image as [registry.access.redhat.com/rhel8/python-{{ spec.short_ver }}](https://access.redhat.com/containers/#/registry.access.redhat.com/rhel8/python-{{ spec.short_ver }}).
 The CentOS7 image is then available on [Docker Hub](https://hub.docker.com/r/centos/python-{{ spec.short_ver }}-centos7/)
```

Comment:
It would be also nice to mention that some Python versions are available on Fedora platform.

Reply:
We can mention Fedora Dockerfiles also in "See also" section.

Reply:
+1

Reply:
Now that I think about it this section should probably be removed completely, else we do not fix #314 ... 
Maybe just pointing to the registries without any image specific links? Something along the lines of:
`RHEL-based images are available in the [Red Hat Registry](https://access.redhat.com/containers])`